_build/pipelines/ci.yml

```diff
@@ -102,8 +102,8 @@ jobs:
         Moniker: WPF
         BuildConfiguration: DebugWPF
         BuildPlatform: Any CPU
-      # Reenable as part of #4441
-      #UWP:
+      # #4441
```

Comment:
Why remove additional context for this issue number?
If all I see is an issue number and I don't remember why it's there (I won't,) if I'm looking at this file and wonder about this commented out section I then have to go and look at the referenced issue to see why that's the case.
Does the referenced issue explain why it's commented out? Does it explain when or why to reenable it? Is it important? Is it just there for information purposes only? Does it directly relate to the commented-out lines below?

I'm all for standardizing formatting and having after the character that indicates a comment, but removing contextual information that is intended to aid in future code maintenance (for the sake of "cosmetic" improvements) doesn't feel like a useful change.

**Not just me but anyone who looks at this file.** - Others are even more likely to need to go and look this up.

Reply:
"Reenable as part of #4441" also isn't enough context to answer the questions you raised. If you think looking at commented lines here is confusing, I'd rather just remove the whole UWP section, but I prefer the way it is now with comments and an issue number for reference.